app/controllers/names_controller.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
 class NamesController < ApplicationController
```

Comment:
Suggestion: 
Disable the cop via a comment on this line (and delete `# rubocop:enable Metrics/ClassLength` at the end of the file).
``` ruby
class NamesController < ApplicationController # rubocop:disable Metrics/ClassLength
```